tests/publiccloud/sles4sap.pm

```diff
@@ -27,15 +37,187 @@ sub upload_ha_sap_logs {
     }
 }
 
+=head2 wait_until_resources_started
+
+    wait_until_resources_started();
+
+Wait for HA cluster to be started.
+=cut
+sub wait_until_resources_started {
```

Comment:
It's too bad this is partially duplicating `hacluster::wait_until_resources_started()`. Ideally, this one should be something similar to a wrapper around the other one, handling cloud-only stuff, and then running `hacluster::wait_until_resources_started()`, but the differences between `script_run` and `$self->{my_instance}->run_ssh_command()` make it difficult to refactor a single function for both.